Formula/enpass-cli.rb

```diff
@@ -0,0 +1,18 @@
+class EnpassCli < Formula
+  desc "Enpass command-line client"
+  homepage "https://github.com/hazcod/enpass-cli"
+  url "https://github.com/hazcod/enpass-cli/archive/refs/tags/v1.6.0.tar.gz"
+  sha256 "25668bb30747dc566695ff7f30e65fa2e29b04f5896155a4ab03185d8f5b4111"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/enpasscli"
+  end
+
+  test do
+    system bin/"enpass-cli", "version"
+    assert_match "enpass-cli arch=", shell_output("#{bin}/enpass-cli version 2>&1")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Thanks for your review! I've made some improvements to tests as you suggested.